backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationResolver.java

```diff
@@ -19,7 +18,7 @@
 	private ApiUserService _userService;
 	private OrganizationService _organizationService;
 
-	public OrganizationResolver(OrganizationService os, ApiUserService users, OrganizationInitializingService initer) {
+    public OrganizationResolver(OrganizationService os, ApiUserService users) {
```

Comment:
Not actually a comment about this PR: would you be open to using an automated code formatter? I got addicted to `go fmt`, `black`, and `dotnet format` and have been using https://github.com/sherter/google-java-format-gradle-plugin on Java projects since.

Reply:
We have an agreement on general formatting conventions and the code is slowly converging toward them, I think? It was messy for a while, and we're slowly digging out. I don't really care for having my code reformatted, so I prefer a fairly light-touch approach, but I've been saying that some day when reasonably few java branches were in flight I would just batch-format the entire source tree and put some harder checkstyle rules in place.

Reply:
Also I have turned on auto-formatting in my IDE and am trying to get used to it, so perhaps I can be trained. 😁